packages/suite-data/files/translations/en.json

```diff
@@ -837,7 +837,6 @@
     "TR_LEARN_ADVANCED_RECOVERY": "Learn how to enter letters",
     "TR_LEARN_AND_DISCOVER": "Suite Guide",
     "TR_LEARN_MORE": "Learn more",
-    "TR_LEARN": "Learn",
```

Comment:
this translation is in use

Reply:
Ah, I see. Check the last commit. Seems some translations were not synced with crowdin yet?

Reply:
tru, fixed in b2e5bf8